docs/swagger.yaml

```diff
@@ -369,6 +369,15 @@ definitions:
       - status
       - address
       - contactPerson
+  PocMerchant:
```

Comment:
I don't think we need to use "PoC" in the spec as long as the whole thing is a PoC.

Reply:
I was unsure if I wanted to delete all the data in the existing contract, but I can do that!